odl/space/numpy_ntuples.py

```diff
@@ -40,13 +40,13 @@
     WeightingBase, MatrixWeightingBase, VectorWeightingBase,
```

Comment:
For the file name, we use in the CUDA case `cu_ntuples.py`. I suggest `npy_ntuples` here as abbreviation, just for consistency. The rename itself is good.